entrypoint.sh

```diff
@@ -47,6 +47,7 @@ exec cloud-api-adaptor-azure azure \
   -region "${AZURE_REGION}" \
   -instance-size "${AZURE_INSTANCE_SIZE}" \
   -resourcegroup "${AZURE_RESOURCE_GROUP}" \
+  -vxlan-port 8472 \
```

Comment:
While the change is good as-is, I wanted to ask you if want to make it configurable across all providers.?

I had to use a similar mechanism for aws when using a ovn cni. I recall @yoheiueda had to use it in ibmcloud as well. So I was planning to add  vxlan-port in the `kustomization.yaml` file, but since you have this PR I wanted to check with you before raising a new PR:-)

Reply:
I think making it configurable among other providers make sense. So should I close this PR?

Reply:
@knrt10 I will need some time before raising a new PR. So let's go ahead with this one so that you are unblocked. 
Hope it's fine ?

Reply:
Cool. Thanks